### ISSUE TITLE
Porting to dotnet core... any interest?

### ISSUE THREAD

**Issue by dealproc** [brockallen/BrockAllen.MembershipReboot#707]
I know there is a good following of folks that have used this library over the years (as have I) and are in the midst of trying to get to dotnet core from netfx.

My question is ... is there enough interest to push an update that is compiled for dotnet core, and if so, what is the preferred netstandard version we should support?

I've done some preliminary work to get the base library over to dotnet core on netstandard 1.3, but am unsure if this is acceptable, or if we should support netstandard 2.0 now that it has [basically] become a thing.

My motivation for doing this is that I'd like to get onto IdSrv 4.0, but without having much time to learn IdSrv 4.0 & ASP.NET Core Identity frameworks both, I'd rather stick with the user management library I use today, and then put a feature request in to build time into my release schedule to port the MR database over to ASP.NET Core Identity.

Thoughts/Comments?

**Comment by brockallen** [comment]
You want the lowest netstandard that is needed to make it work.

**Comment by dealproc** [comment]
Right... obviously... the difference is with netstandard 1.3; I have a good list of libraries i had to include that seems to be within netstandard 2.0 out of the box.

With things like IdentityServer 4.0 going to netstandard 2.0, etc., wasn't sure if it was wortwhile to follow in-step; or still stick with the "lowest version possible" mantra.

**Comment by brockallen** [comment]
Well, you have a point... so if netstandard2 has it all, then use it.
